plugins/modules/content_import_info.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2024, Partha Aji
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: content_import_info
+version_added: 4.0.0
+short_description: List content imports
+description:
+    - List information about content imports.
+author:
+    - "Partha Aji (@parthaa)"
+options:
+  id:
+    description:
+      - Export history identifier.
+    required: false
+    type: int
+  content_view_version:
+    description:
+      - Content view version.
+    required: false
+    type: str
+  content_view:
+    description:
+      - Content view name.
+    required: false
+    type: str
+  type:
+    description:
+      - Specify complete or incremental imports.
+    required: false
+    type: str
+    choices:
+    - complete
+    - incremental
+extends_documentation_fragment:
+  - theforeman.foreman.foreman
+  - theforeman.foreman.foreman.katelloinfomodule
+  - theforeman.foreman.foreman.infomodulewithoutname
+'''
+
+EXAMPLES = '''
+- name: "List all imports in the organization"
+  theforeman.foreman.content_import_info:
+    organization: "Default Organization"
+    type: complete
+    username: "admin"
+    password: "changeme"
+    server_url: "https://foreman.example.com"
+- name: "Get a specific import history and register the result for the next task"
+  vars:
+    organization_name: "Export Org"
+  theforeman.foreman.content_import_info:
+    id: 29
+    username: "admin"
+    password: "changeme"
+    server_url: "https://foreman.example.com"
+    organization: "Default Organization"
+  register: result
+- name: "List all imports of a specific content view version"
+  theforeman.foreman.content_import_info:
+    content_view: RHEL8
+    content_view_version: '1.0'
+    username: "admin"
+    password: "changeme"
+    server_url: "https://foreman.example.com"
+    organization: "Default Organization"
+- name: "List incremental imports of a specific content view version"
+  theforeman.foreman.content_import_info:
+    content_view: RHEL8
```

Comment:
This doesn't actually specify a version.
```suggestion
- name: "List incremental imports of a specific content view"
  theforeman.foreman.content_import_info:
    content_view: RHEL8
```
(I think this bug is also in the one you copied from ;)